src/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { Tab } from "@headlessui/react";
+
+type TabItem = {
+  label: string;
+  content: React.ReactNode; // or JSX.Element
+};
+
+type TabsProps = {
+  tabs: TabItem[];
+};
+
+function Tabs({ tabs }: TabsProps) {
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  return (
+    <Tab.Group
+      selectedIndex={selectedIndex}
+      onChange={setSelectedIndex}
+      defaultIndex={0}
+    >
+      <Tab.List className="mb-[20px] max-w-[354px] w-full h-[40px] flex">
```

Comment:
Looks good, mb-[20px] can be replaced with mb-5 and h-[40px] can be replaced with h-10